src/types/index.ts

```diff
@@ -20,6 +20,11 @@ export interface FsNode {
   type: FsNodeType;
 }
 
+export interface FsChild {
+  filename: string;
```

Comment:
actually, do you mind changing this to `path` (`FSNode`) and patching on replit-web by appending the watched directory's name? at least in the extension adapter code not `watchDir` on the fs service as a first pass.

sorry for churn, i think the way you had it is more intuitive, it also reduce API surface area and interfaces.

Reply:
cc @moudy you probably used this API the most, do you think `FSChild` is a good idea for `fs.watchDir` change events

Reply:
I'm sorry, I don't exactly understand what needs to be done here.  Will discuss with you later today in the office.

Reply:
I would prefer `FSNode`. I often find that I need the pull path